### PR TITLE
CQF-522 move EXM124 fhir401 to release and add its bundle from the

### PR DESCRIPTION
**Pull request by matthewtiller**
Connectathon repo

### CODE REVIEW

**Comment by matthewtiller** [comment]
This pull request is to move EXM124 fhir401 to "release" and add its bundle from the Connectathon repo

**Comment by matthewtiller** [comment]
Medium priority for Sept Connectathon

File: bundles/release/fhir401/EXM124-9.0.000/EXM124-9.0.000-bundle.json [hunk 1]
```diff
@@ -0,0 +1,4208 @@
...
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/library-type",
+              "code": "logic-library"
+            }
+          ]
+        },
+        "relatedArtifact": [
+          {
+            "type": "depends-on",
+            "resource": "http://fhir.org/guides/dbcg/connectathon/Library/library-MATGlobalCommonFunctions|5.0.000"
```
Comment (okeefm) [inline bundles/release/fhir401/EXM124-9.0.000/EXM124-9.0.000-bundle.json:317]: Is this the actual canonical URL that should reference the `Library` resource for `MATGlobalCommonFunctions`? The actual `url` attribute of `MATGlobalCommonFunctions-FHIR` in `resources/library/` is `http://hl7.org/fhir/us/cqfmeasures/Library/library-mat-global-common-functions-FHIR`, and the version is `2.0.000`. There is a `MATGlobalCommonFunctions-5.0.000` Library in the `library-deps` bundle included in this PR, but that bundle doesn't have a canonical URL.

File: bundles/release/fhir401/EXM124-9.0.000/EXM124-9.0.000-bundle.json [hunk 2]
```diff
@@ -0,0 +1,4208 @@
...
+          },
+          {
+            "type": "depends-on",
+            "resource": "http://fhir.org/guides/dbcg/connectathon/Library/library-FHIRHelpers|4.0.1"
```
Comment (okeefm) [inline bundles/release/fhir401/EXM124-9.0.000/EXM124-9.0.000-bundle.json:321]: Same note as above, with `FHIRHelpers|4.0.1` substituted for `MATGlobalCommonFunctions|5.0.000`

**Pull request merged**